modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -56,16 +56,17 @@ class Instrument_List extends \NDB_Menu_Filter
     {
         // create user object
         $user =& \User::singleton();
-
+        $candidate =& \Candidate::singleton($_REQUEST['candID']);
+        $timePoint =& \TimePoint::singleton($_REQUEST['sessionID']);
         // check user permissions
         return ($user->hasPermission('access_all_profiles')
                 || (in_array(
-                    $this->candidate->getData('CenterID'),
+                    $candidate->getData('CenterID'),
```

Comment:
These variables are initialized on lines 101-113. If that's not working, it needs to be investigated.. using superglobals instead of the router is a step backwards.